src/app/work-item/work-item-detail/markdown-control/markdown-control.component.html

```diff
@@ -54,6 +54,7 @@
   <div class="col-md-12 col-sm-12 col-xs-12 detail-text-action">
       <span class="col-md-4">
         <small *ngIf="loggedIn && tabBarVisible">
+          <img src="../../../../assets/img/markdown-solid.png" class="markdown-mark" />
           <b>Markdown Supported</b>
```

Comment:
that's odd. locally it worked and built w/o issue. maybe a symlink is needed? just not sure what it would be